timelock-agent/src/main/java/com/palantir/timelock/paxos/ClientAwarePaxosLearner.java

```diff
@@ -48,19 +47,16 @@
     @Consumes(MediaType.APPLICATION_JSON)
     void learn(@PathParam("client") String client, @PathParam("seq") long seq, PaxosValue val);
 
-    @Nullable
     @GET
     @Path("learned-value/{seq}")
     @Produces(MediaType.APPLICATION_JSON)
-    PaxosValue getLearnedValue(@PathParam("client") String client, @PathParam("seq") long seq);
+    Optional<PaxosValue> safeGetLearnedValue(@PathParam("client") String client, @PathParam("seq") long seq);
```

Comment:
this needn't be called `safeGetLearnedValue` since this doesn't implement `PaxosLearner`

Reply:
makes sense